R/civis_future.R

```diff
@@ -0,0 +1,167 @@
+#' Evaluate an expression in Civis Platform
+#'
+#' @param ... Arguments to \code{\link{CivisFuture}}
+#' @return The result of evaluating \code{expr}.
+#'
+#' @examples \dontrun{
+#'  eval_civis(quote(sqrt(10)))
+#' }
+#'
+#' @export
+civis_platform <- function(...) {
+  future <- CivisFuture(...)
+  if (!future$lazy) future <- run(future)
```

Comment:
where does the function `run`  come from?  Does there need to be a `::` or an import?

Reply:
Ah, okay I see this in L72

Reply:
I'll just move this to the top for clarity